### ISSUE TITLE
loop

### ISSUE THREAD

**Issue by tobydp** [dmainmon/Geiger-Counter-IoT#1]
hey.

i try to run your geiger sketch. i flash a esp8266 nodemcu 1.0 with your sketch and int get a loop on the device.

`ets Jan  8 2013,rst cause:2, boot mode:(3,6)

load 0x4010f000, len 3460, room 16 
tail 4
chksum 0xcc
load 0x3fff20b8, len 40, room 4 
tail 4
chksum 0xc9
csum 0xc9
v0004f390
~ld
`

have you observed this behaviour before? have you reflashed it in the past years? many thanks

**Comment by KristyGourab** [comment]
Any solution?

**Comment by tobydp** [comment]
No, unfortunately not. Have it then no longer pursued and built a Multigeiger. https://github.com/ecocurious2/MultiGeiger Maybe something for you.
